concourse/scripts/kibana_set_utc.rb

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env ruby
+
+require 'net/http'
+require 'json'
+
+def ingest_to_es(document, uri)
+  raise "Document is not a hash" unless document.is_a? Hash
+  Net::HTTP.new(uri.host, uri.port).start do |http|
+    req = Net::HTTP::Put.new(uri.path, 'Content-Type' => 'application/json')
+    req.body = document.to_json
+    http.request(req)
+  end
+end
+
+def date_format_not_utc(response_json)
+end
+
+es_host = ENV.fetch("ES_HOST").delete(' ')
+es_port = ENV.fetch("ES_PORT").delete(' ')
+
+es_url = "http://#{es_host}:#{es_port}"
+date_format_url = "#{es_url}/.kibana/config/4.3.1"
+date_format_uri = URI(date_format_url)
+
+response = JSON.parse(Net::HTTP.get(date_format_uri))
+
+if ! response["found"] || response["_source"]["dateFormat:tz"] != "UTC"
```

Comment:
> This silently fails to set the timezone if there's a problem connecting to ES or the index doesn't exist. 

`response = JSON.parse(Net::HTTP.get(date_format_uri))` will fail non-silently if we can't connect to ES. If index is not found, which is perfectly fine, ES will return 404 code and json document in the response body containing `"found": false`, in such case script will create `.kibana` index with `dateFormat:tz` setting.

> Are there any cases where we would want it to proceed on failure?

In my opinion there are no such cases. If we can't update ( or create ) index we shouldn't continue. 

>Is the index only created the first time you log in to Kibana?

Yes, but it can be updated later to store dashboards etc. If it already exists it is (re)used.

> We should also have some basic Aruba tests for this script

I'll look into this.